### PR TITLE
fix(pat navigationmarker): Only mark the 'Home' entry in nav when at root

### PR DESCRIPTION
**Pull request by frapell**

### CODE REVIEW

**Comment by frapell** [comment]
@jenkins-plone-org please run jobs

**Comment by frapell** [comment]
@thet Related to https://github.com/plone/mockup/issues/1164

File: src/pat/navigationmarker/navigationmarker.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export default Base.extend({
     parser: "mockup",
     init: function () {
         var self = this;
+        var portal_url = $('body').data('portal-url');
```
Comment (thet) [inline src/pat/navigationmarker/navigationmarker.js:10]:
```suggestion
        const portal_url = document.body.dataset.portalUrl;
```

File: src/pat/navigationmarker/navigationmarker.js [hunk 2]
```diff
@@ -34,6 +35,10 @@ export default Base.extend({
                         inPath = true;
                     }
                 }
+                if ((navlink == portal_url) && (href != portal_url)) {
```
Comment (thet) [inline src/pat/navigationmarker/navigationmarker.js:38]:
```suggestion
                if ((navlink === portal_url) && (href !== portal_url)) {
```

**Review by thet — APPROVED** [review]
LGTM. just a minor suggestion to avoid jQuery and use tripple comparison operators, as normally suggested.

**Comment by frapell** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**